src/app/launcher/create-app/mission-runtime-createapp-step/mission-runtime-createapp-step.component.spec.ts

```diff
@@ -120,4 +120,228 @@ describe('MissionRuntimeStepComponent', () => {
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionGroupItem = missionsSection.querySelector('.list-group-item');
+    let radioBtn = <HTMLInputElement>missionGroupItem.querySelector('input[type="radio"]');
+    let beforeChange = missionGroupItem.classList.contains('selected-list-item');
+    radioBtn.click();
+    tick();
+    fixture.detectChanges();
+    expect(component.launcherComponent.summary.mission).toBe(component.missions[0]);
+  }));
+
+  it('should have the suggested missions tag when mission.suggested field is true', () => {
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let featuredTag = missionsSection.querySelector('.f8launcher-tag-featured');
+    expect(featuredTag).toBeDefined();
+  });
+
+  it('should not have the suggested missions tag when mission.suggested field is false', () => {
+    component.missions[0].suggested = false;
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let featuredTag = missionsSection.querySelector('.f8launcher-tag-featured');
+    expect(featuredTag).toBeNull();
+  });
+
+  it('should have the mission name as specified', () => {
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionHead = missionsSection.querySelectorAll('.list-group-item-heading');
+    let missions = component.missions, len = missions.length;
+    for (let i = 0; i < len; ++ i) {
+      expect((<HTMLDivElement>missionHead[i]).innerText).toBe(missions[i].name);
+    }
+  });
+
+  it('should have the mission description as specified', () => {
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let missionHead = missionsSection.querySelectorAll('.list-group-item-text');
+    let missions = component.missions, len = missions.length;
+    for (let i = 0; i < len; ++ i) {
+      expect((<HTMLDivElement>missionHead[i]).innerText).toBe(missions[i].description);
+    }
+  });
+
+  it('should show more if the url is present', () => {
+    component.missions[0].url = 'https://github.com/fabric8-launcher/ngx-launcher/';
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let showMore = missionsSection.querySelector('a');
+    expect(showMore).toBeDefined();
+  });
+
+  it('should have the mission.url as href', () => {
+    component.missions[0].url = 'https://github.com/fabric8-launcher/ngx-launcher/';
+    fixture.detectChanges();
+    element = fixture.nativeElement;
+    let missionsSection = element.querySelectorAll('.card-pf-body')[0];
+    let showMore = <HTMLAnchorElement>missionsSection.querySelector('a');
+    expect(showMore.href).toBe('https://github.com/fabric8-launcher/ngx-launcher/');
```

Comment:
Instead of comparing the URL, perhaps expect(showMore).toBeDefined() would be enough? I'm seeing this error: Expected 'javascript:void(0)' to be 'https://github.com/fabric8-launcher/ngx-launcher/'.

You're obtaining a link, but the retrieved anchor is used to show/hide truncated text.